package.json

```diff
@@ -36,7 +36,7 @@
     "zone.js": "^0.6.23"
   },
   "devDependencies": {
-    "@angular/cli": "^1.0.0",
+    "@angular/cli": "^1.0.2",
```

Comment:
one thing is we could have a customised plan on upgrades. A day in a month or so. where we document the errors we face on upgrading.
(or) something like this in an ad-hoc way?
I prefer the first, what do you all suggest?
@Marauderer97 @harshit98 @isuruAb @mariobehling